opencti-platform/opencti-front/src/private/components/workspaces/Workspaces.tsx

```diff
@@ -0,0 +1,130 @@
+import React, { FunctionComponent } from 'react';
+import {
+  WorkspacesFilter,
+  WorkspacesLinesPaginationQuery,
+  WorkspacesLinesPaginationQuery$variables,
+} from '@components/workspaces/__generated__/WorkspacesLinesPaginationQuery.graphql';
+import { WorkspaceLineDummy } from '@components/workspaces/WorkspaceLine';
+import ListLines from '../../../components/list_lines/ListLines';
+import WorkspacesLines, { workspacesLinesQuery } from './WorkspacesLines';
+import WorkspaceCreation from './WorkspaceCreation';
+import Security from '../../../utils/Security';
+import { EXPLORE_EXUPDATE } from '../../../utils/hooks/useGranted';
+import { usePaginationLocalStorage } from '../../../utils/hooks/useLocalStorage';
+import useQueryLoading from '../../../utils/hooks/useQueryLoading';
+
+interface WorkspacesProps {
+  type: string;
+}
+
+const Workspaces: FunctionComponent<WorkspacesProps> = ({
+  type,
+}) => {
+  const {
+    viewStorage,
+    paginationOptions,
+    helpers: storageHelpers,
+  } = usePaginationLocalStorage<WorkspacesLinesPaginationQuery$variables>(
+    `view-${type}-list`,
+    {
+      searchTerm: '',
+      sortBy: 'name',
+      orderAsc: false,
+      openExports: false,
+      redirectionMode: 'overview',
+      view: 'lines',
+    },
+  );
+
+  const {
+    numberOfElements,
+    searchTerm,
+    sortBy,
+    orderAsc,
+  } = viewStorage;
+
+  const workspacePaginationOptions = { ...paginationOptions, filters: [{ key: ['type' as WorkspacesFilter], values: [type] }] };
```

Comment:
@Archidoit we'll need to change that after this PR merge and  a  rebase on our side.

Reply:
right I keep this in mind :)